velox/type/TimestampConversion.cpp

```diff
@@ -273,7 +273,7 @@ bool tryParseDateString(
 
   // No month or day.
   if ((mode == ParseMode::kSparkCast || mode == ParseMode::kIso8601) &&
-      pos == len) {
+      (pos == len || buf[pos] == 'T')) {
```

Comment:
is the additional `T` check also valid for kSparkCast and also fixing it? If yes, should we tag someone working on the Spark cast to add relevant test cases?

Reply:
Yes, according to the docs, SparkCast should be the same as Iso8601, just that it additionally allows trailing or leading spaces.

Most of this code predates when Velox was open sourced, maybe @marin-ma has worked on it?